docs/plans/2026-03-29-pr10-bugfixes.md

```
@@ -0,0 +1,402 @@
+
+> **For Claude:** REQUIRED SUB-SKILL: Use superpowers:executing-plans to implement this plan task-by-task.
+
+**Goal:** Fix all bugs and review findings from PR #10 (Camera FAB + global upload flow + provider-agnostic OCR).
+
+**Architecture:** Targeted fixes across API (Python + TypeScript) and web (React). No new features — just fixing what's broken and cleaning up what's sloppy.
+
+**Tech Stack:** Python (reportlab), TypeScript (Node/Express), React (Next.js), Docker Compose, Caddy
+
+---
+
+### Task 1: Remove secrets from `.env.docker`
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)